### ISSUE TITLE
select method not working ?

### ISSUE THREAD

**Issue by mlarcher** [romgain/react-select-event#18]
React-select-event comes in very handy, I've been able to take advantage of it to open the menu and make assertions on it and it helped a lot. Still, when trying to `select` something, I get the following error ` Unable to find an element with the text: /termvalue1/. This could be because the text is broken up by multiple elements. In this case, you can provide a function for your text matcher to make your matcher more flexible`, even though the element is present in the DOM and visible with a debug(). After digging a while into the code, it appeared that the error comes from https://github.com/romgain/react-select-event/blob/master/src/index.ts#L70 . commenting this line out makes the test behave as expected.
Is it a regression or is it the expected behaviour and I'm missin gsomething ?

**Comment by romgain** [comment]
Hi @mlarcher !

We did this explicitly to filter out the "aria-context" which would sometimes be targeted in addition to the actual select item.

Could you please share some code that reproduces this issue? (Or PR in a failing test case that shows the issue if you can!)

**Comment by mlarcher** [comment]
Sorry I don't have time to setup a working example. The `select`  function is small enough that I took what I needed out of it and used it in my test. 
If you want to dig it further, maybe the component can be a starting point... Sorry I can't do more for now
```js
<AsyncSelect
        loadOptions={loadOptions}
        onChange={handleChange}
        isMulti
        defaultOptions
        cacheOptions
        formatOptionLabel={formatOptionLabel}
        value={selectedSkills}
      />
```

**Comment by romgain** [comment]
Unfortunately there isn't much I can do without more information.

My guess is that `formatOptionLabels` could be the cause of the issues you're having, but without any meaningful code I don't see what I can do...

**Comment by mlarcher** [comment]
You are correct in your assumption : if I remove `formatOptionLabel`, react-select-event works fine 👏 
I don't know if the issue is on react-select side or on your side, but even when formatOptionLabel is a simple function that returns `<div>{label}</div>`, the `select`  method fails too.
You can experiment with it by passing `formatOptionLabel={({ label }) => <div>{label}</div>}` on the AsyncSelect component.

**Comment by romgain** [comment]
Cool, thank you @mlarcher !

This should now be fixed in v4.1.3!
Let me know if that works for you!
